typeDefs.js

```diff
@@ -201,6 +276,16 @@ const typeDefs = gql`
     deleteUser(id: ID!): User
     updateUserById(user: UserInput!, id: ID!): User
     updateUserByToken(user: UserInput!): User
+
+    "Products Mutation"
+    addProducts(product: productsInput!): Products
+    deleteProducts(id: ID!): Products
+    updateProductById(id: ID!, product: productsInput!): Products
+
+    "Comments Mutation"
+    addComments(comment: commentsInput!): Comments
+    deleteCommentsById(id: ID!): Products
```

Comment:
Please, rename it to `deleteComment`. I don't think that we need `byId` in the end because arguments definition already shows that it required id

Reply:
done